.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: Build Docker image
```

Comment:
Could we add a comment to this workflow that states its purpose? As I understand it, the purpose of this workflow is to catch any pull request that breaks the Docker image (so that we are spared the unpleasantness of discovering during a release that somebody broke it).